tools/goctl/api/gogen/handler.tpl

```diff
@@ -1,26 +1,27 @@
-package {{.PkgName}}
```

Comment:
Do not modify template variables to prevent incompatibility problems for users who customize templates.

Reply:
ok,  new record has been submitted.  @kesonan